code/modules/overmap/ships/computers/helm.dm

```diff
@@ -43,6 +45,14 @@ var/global/list/overmap_helm_computers
 
 /obj/machinery/computer/ship/helm/Process()
 	..()
+
+	if (current_operator)
+		if (!linked)
+			to_chat(current_operator, SPAN_DANGER("\The [src]'s controls lock up with an error flashing across the screen: Connection to vessel lost!"))
+			set_operator(null, TRUE)
+		if (!Adjacent(current_operator) || CanUseTopic(current_operator) != STATUS_INTERACTIVE || !viewing_overmap(current_operator))
```

Comment:
Was this meant to be an `else if` or is there a missing return earlier?

Reply:
That should in fact be an `else if`